batch2/create-batch-tables.sql

```diff
@@ -32,7 +32,10 @@ CREATE TABLE IF NOT EXISTS `batches` (
   `n_succeeded` INT NOT NULL DEFAULT 0,
   `n_failed` INT NOT NULL DEFAULT 0,
   `n_cancelled` INT NOT NULL DEFAULT 0,
-  `time_created` TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+  `time_created` DOUBLE NOT NULL,
+  `time_completed` DOUBLE,
+  `msec_mcpu` BIGINT NOT NULL DEFAULT 0,
+  `cost` DOUBLE NOT NULL DEFAULT 0
```

Comment:
`msec_mcpu` is just units, I'd call it `cost_msec_mcpu`.  I wouldn't keep two costs, just convert msec_mcpu when you read it out.

There is another issue, that we're not calculating the cost of the disk.  To start, I think we can keep it as 100G and fold it into the per core-time cost.  That means larger disk  sizes won't be accounted for.  I think that's OK for now.  We can track the disk cost directly when we support requested disk sizes.

Reply:
I was concerned that if google's pricing changed some day (or we change our model), then we'd be using the new rate when calculating the cost of older batches. That's why I thought we needed both the amount of resources used and the cost computed from the total number of resources used.

Reply:
I wouldn't worry about that now.  I think this (and full and accurate resource tracking with costs) is going to require a more complicated solution.